src/EventStore.Core/TransactionLog/Chunks/TFChunkScavenger.cs

```diff
@@ -76,6 +76,21 @@ private IEnumerable<TFChunk.TFChunk> GetAllChunks(int startFromChunk)
             }
         }
 
+        public Task MergeIndexes()
```

Comment:
As mentioned [here](https://github.com/EventStore/EventStore/pull/1786/files#r244915589), it would be better to move this to `IndexCommiterService.cs` instead of `TFChunkScavenger.cs`